File: ts/iq/waffle~get~timestamp.ts
/** <iq type="get" xmlns="waffle">. Replies with one of: Success, Error. */
export function buildWaffleForceSuspendState(smaxId: number): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'waffle', smax_id: smaxId, to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'timestamp', attrs: {  } },
    ] }
}

File: go/iq/waffle~get~timestamp.go
// BuildWaffleForceSuspendState builds <iq type="get" xmlns="waffle">. Replies with one of: Success, Error.
func BuildWaffleForceSuspendState(smaxId int) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "waffle", "smax_id": smaxId, "to": "s.whatsapp.net", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "timestamp", Attrs: waBinary.Attrs{}},
		},
	}
}